lib/tarfiles.py

```diff
@@ -92,8 +92,25 @@ def do_tarballs(args: argparse.Namespace) -> None:
     we convert the argument to the next type as we go...
     """
     # pylint: disable=too-many-nested-blocks,too-many-branches
-    res = []
     clean_up: List[str] = []
+    res: List[str] = []
+    path: Optional[str] = None
+    for fn in args.input_file:
+        if fn.startswith("dropbox:"):
+            tarfile = tar_up(
+                os.path.dirname(fn[8:]), "/dev/null", os.path.basename(fn[8:])
```

Comment:
I've never been a fan of substringing like this (`fn[8:]`) in these cases, just because it makes the reader have to do extra mental work to figure out what's at position 8, and therefore what the resultant string would be.  What would you think about using replace (e.g. fn.replace("dropbox:", "") or a regex?  If you agree, I'll put in an issue to do the same elsewhere in the code (for example, on line 117 for `tardir:`).